src/compiler/program.ts

```diff
@@ -559,6 +559,12 @@ namespace ts {
         sourceMap: false,
     };
 
+    interface OutputFingerprint {

```

Comment:
Whoops! Misread the code, forgive me. I was taking a quick glance at this PR again since it's been a bit.